data/input/source/MacAddresses/mac-vendor.csv

```diff
@@ -251,7 +251,6 @@ MA-L,D821DA,SERNET (SUZHOU) TECHNOLOGIES CORPORATION,"NO.8 Tangzhuang Road,Suzho
 MA-L,DCAC6F,Everytale Inc,2093 Philadelphia Pike #2211 Claymont DE US 19703 
```

Comment:
I see that sorting has changed. Should we add sorting here?

Reply:
For now it doesn't make sense